src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -462,6 +462,10 @@
 
     <!-- Include SonarLint assemblies, the SonarQube.Client library, and the analyzers-->
     <AsmRefIncludePatterns>Sonar*</AsmRefIncludePatterns>
+    <!-- SonarSecrets is referenced as a released package, so it is always signed, regardless of the value of
+         SignAssembly. This messes up the comparison of unsigned assemblies we do below, causing the AsmRef
+         comparison to fail. For now, we'll expclude it from the comparison. -->
+    <AsmRefExcludePatterns>!SonarLint.Secrets.DotNet*</AsmRefExcludePatterns>
```

Comment:
This was an additional change I had to stop the AsmRef check from failing the build.
I'll add an `Infrastructure` ticket to see if we can re-include it.